core/src/main/java/org/opensearch/sql/executor/ExecutionEngine.java

```diff
@@ -53,6 +54,8 @@ void execute(PhysicalPlan plan, ExecutionContext context,
   class QueryResponse {
     private final Schema schema;
     private final List<ExprValue> results;
+    private final long total;
```

Comment:
What is `total` for? Total pages?

Reply:
Total hits in OpenSearch terms (number of docs matched search criteria) = `total` in SQL response (number of rows returned).